app/controllers/accounts/history_controller.rb

```diff
@@ -13,6 +13,7 @@ def show
         sp_name: decorated_session.sp_name,
         decorated_user: current_user.decorate,
         locked_for_session: pii_locked_for_session?(current_user),
+        account_reset_requested: current_user&.account_reset_request,
```

Comment:
ditto for boolean https://github.com/18F/identity-idp/pull/6513#discussion_r910307525

```suggestion
        account_reset_requested: current_user&.account_reset_request.present?,
```